CMakeLists.txt

```diff
@@ -63,7 +64,6 @@ add_message_files(
    FlowStamped.msg
    FlowArray.msg
    FlowArrayStamped.msg
-   RectArrayStamped.msg
```

Comment:
Why? You duplicate this definition on purpose? https://github.com/ros-perception/opencv_apps/pull/63/files/7d1fbf69ef4865f28917adc0ba7703973dd8926c#diff-af3b638bc2a3e6c650974192a53c7291R62